### ISSUE TITLE
Restore Packages command doesn't work for .NET Framework projects

### ISSUE THREAD

**Issue by adamhartford** [OmniSharp/omnisharp-vscode#1507]
## Environment data
`dotnet --info` output:
```
.NET Command Line Tools (1.0.4)

Product Information:
 Version:            1.0.4
 Commit SHA-1 hash:  af1e6684fd

Runtime Environment:
 OS Name:     Mac OS X
 OS Version:  10.12
 OS Platform: Darwin
 RID:         osx.10.12-x64
 Base Path:   /usr/local/share/dotnet/sdk/1.0.4
```
VS Code version: 1.12.2
C# Extension version: 1.9.0

## Steps to reproduce

1. `$ dotnet new console`
2. `$ code .`
3. In csproj, change TargetFramework to `<TargetFramework>net452</TargetFramework>`
4. Choose `.NET: Restore Packages` from the command palette.

## Expected  behavior

`dotnet restore` is executed and packages are restored.

## Actual behavior

Error message stating: `No .NET Core projects found`.

## Note

I have a pull request ready to fix this, but is the current behavior by design?

**Comment by DustinCampbell** [comment]
This fix will appear in our first 1.11 beta.

**Comment by DustinCampbell** [comment]
Thanks again for contributing!
